src/space_flight/pipelines/data_engineering/nodes.py

```diff
@@ -101,3 +101,37 @@ def create_master_table(
     master_table = with_companies.drop(["shuttle_id", "company_id"], axis=1)
     master_table = master_table.dropna()
     return master_table
+
+
+# Implementation of a sorting algorithm
+# Source: https://bosker.wordpress.com/2011/04/29/the-worst-algorithm-in-the-world/
+def _slowsort(values, i, j):
```

Comment:
i think `i` and `j` are not clear. best practice will be to describe them in context like `index` or, i dont know what `j` means

Reply:
Okay, can you suggest what i and j could be?

should i rename i to index?

Reply:
Yes, i'll rename as discussed, thanks for hte feedback! <3 

![great success](https://user-images.githubusercontent.com/58553/191441036-2c7db92f-bb38-43db-9813-b32af3585af8.png)

Reply:
i am guessing `j` would mean `jump`? is that correct?